tests/integration/test_charm.py

```diff
@@ -175,3 +175,23 @@ async def _change_cronjob_to_minutes(unit: Unit, current_hour_interval: int):
     cron_content = await unit.ssh(command=f"cat {CRON_BUILD_SCHEDULE_PATH}")
     logger.info("Cronfile content: %s", cron_content)
     await unit.ssh(command="sudo systemctl restart cron")
+
+
+# This test implicitly assumes that it is run after an image-builder run which produces logs.
```

Comment:
Instead of relying on the image builder run, you could write or append something manually to these files. Generally, tests shouldn't rely on each other, but for heavy integration tests, relying on each other can sometimes make sense to speed things up. In this case, however, I think we could switch so that we don't rely on the run, especially since the test doesn't assert that certain log lines exist; it just looks for logrotate lines.

Reply:
Great idea, thanks! Updated :)